HPXMLtoOpenStudio/resources/lighting.rb

```diff
@@ -308,7 +310,7 @@ def self.get_schedule(model, weather)
         normalized_hourly_lighting[month][hour] = ltg_hour / sum_kWh
         monthly_kwh_per_day[month] = sum_kWh / 2.0
       end
-      wtd_avg_monthly_kwh_per_day += monthly_kwh_per_day[month] * days_m[month] / 365.0
+      wtd_avg_monthly_kwh_per_day += monthly_kwh_per_day[month] * days_m[month] / num_days_in_year
```

Comment:
This doesn't seem correct if I'm understanding the change. A year with a leap day should have increased lighting energy use compared to a year without a leap day.

Reply:
What is the behavior for other end uses (appliances, plug loads, etc.)?

Reply:
@shorowit I'm showing that the lighting energy use remains the same between non leap year and leap year. And the way lighting energy use is currently calculated, this makes sense doesn't it? Isn't lighting energy use a function of floor area only?

Perhaps we could bump the value up for leap years by adjusting the usage multiplier, which would essentially scale up the [regression coefficients](https://github.com/NREL/OpenStudio-HPXML/blob/build-res-hpxml-v3-schedule-files/HPXMLtoOpenStudio/resources/lighting.rb#L212)?

Reply:
We found that schedules were improperly handling leap years. That fix is captured in a separate PR.